sys/net/gnrc/link_layer/lorawan/gnrc_lorawan_region.c

```diff
@@ -44,7 +45,16 @@ int gnrc_lorawan_set_dr(gnrc_lorawan_t *mac, uint8_t datarate)
 
 uint8_t gnrc_lorawan_rx1_get_dr_offset(uint8_t dr_up, uint8_t dr_offset)
 {
+#if (IS_ACTIVE(CONFIG_LORAMAC_REGION_EU_868))
```

Comment:
as discussed offline, this should be it's own function for every region:
```c
#if (IS_ACTIVE(CONFIG_LORAMAC_REGION_EU_868))
uint8_t gnrc_lorawan_rx1_get_dr_offset(uint8_t dr_up, uint8_t dr_offset)
{
...
}
#endif
```

Otherwise we cannot split these files later

Reply:
Sure.